### PR TITLE
Joss2

### PR DESCRIPTION
**Pull request by MaceKuailv**
Paper fixes in response to editor's comments
@ThomasHaine What do you think?

### CODE REVIEW

**Comment by MaceKuailv** [comment]
@ThomasHaine Please take a look at the changes to paper/paper.md. If you think it is fine, feel free to merge this PR.

File: paper/paper.md [hunk 1]
```diff
@@ -46,15 +47,15 @@ While some usage examples are presented here, many more can be found in the docu
...
+As an example of seaduck's interpolation/regridding functionality, consider a realistic simulation of the Kangerdlugssuaq Fjord, which is in east Greenland [@Fraser2018]. This is an MITgcm [@mitgcm] simulation with uneven grid spacing such that grid cells within the fjord are much more densely packed than elsewhere. The goal is to interpolate, and hence regrid, the sea surface height field, $\eta$, to a uniform grid spacing in the southern part of the domain. In Figure. 1, the coherent patch between 66.5 N and 67 N is a very dense scatter plot of the interpolated value where neighboring points are connected together. The rest of this plot is scatter plot of model output at center grid points. The close agreement between the interpolated and output value can be clearly seen in Figure. 1. The interpolation also remains smooth near strong gradient and land boundaries.
```
Comment (ThomasHaine) [inline paper/paper.md:50]: Line 50: I suggest "As an example of seaduck's interpolation/regridding functionality, consider a realistic simulation of the Kangerdlugssuaq Fjord, which is in east Greenland [@Fraser2018]. The simulation uses the MITgcm [@mitgcm] with uneven grid spacing such that grid cells within the fjord are much more densely packed than elsewhere. The goal is to interpolate, and hence regrid, the sea surface height field, $\eta$, to a uniform grid spacing in the southern part of the domain. In Figure. 1, the coherent patch between 66.5 N and 67 N is a very dense scatter plot of the interpolated $\eta$ field.  The close agreement between the interpolated and output value can be clearly seen in Figure. 1. The interpolated field also remains smooth near strong gradients and land boundaries."

File: paper/paper.md [hunk 2]
```diff
@@ -46,15 +47,15 @@ While some usage examples are presented here, many more can be found in the docu
...
+In this example, a stationary, surface slice of the LLC4320 [@llc4320] simulation is used. LLC4320 is a kilometer-scale model of the global ocean circulation with complex topology. 150,000 Lagrangian particles are released randomly and evenly on the globe, and seaduck computes their trajectories for 30 days. Figure. 2 shows the particle trajectories for the northern hemisphere, which contains around 10$^8$ velocity points. The colors denote the current speed. This simulation takes about an hour to run on SciServer [@sciserver].
 
-![Fig 2. Streaklines of particle advected by stationary 2D slice of the LLC4320 simulation. Colors denote the current speed.](https://github.com/MaceKuailv/seaduck_sciserver_notebook/blob/master/stable_images/LLC4320_29_2.png?raw=true)
+![Streaklines of particle advected by stationary 2D slice of the LLC4320 simulation. Colors denote the current speed.](https://github.com/MaceKuailv/seaduck_sciserver_notebook/blob/master/stable_images/LLC4320_29_2.png?raw=true)
```
Comment (ThomasHaine) [inline paper/paper.md:58]: Change "Streaklines of particle" to "Particle streaklines".

**Review by ThomasHaine — CHANGES_REQUESTED** [review]
I made a few comments on the text. Unless you have questions, implement the changes and merge the PR.

**Comment by MaceKuailv** [comment]
@ThomasHaine I changed all of them, I am going to merge this shortly, and test if the http issue is resolved or not.

**Pull request merged**